packages/sanity/src/core/form/inputs/ReferenceInput/useReferenceInput.tsx

```diff
@@ -113,8 +115,13 @@ export function useReferenceInput(options: Options) {
   }, [disableNew, initialValueTemplateItems, schemaType.to])
 
   const getReferenceInfo = useCallback(
-    (id: string) => adapter.getReferenceInfo(documentPreviewStore, id, schemaType),
-    [documentPreviewStore, schemaType],
+    (id: string) =>
+      adapter.getReferenceInfo(documentPreviewStore, id, schemaType, {
+        version: perspective?.startsWith('bundle.')
```

Comment:
nit pick:

Could we create a reusable `getBundleFromPerspective` function?
```
 function getBundleFromPerspective(perspective: string) {
  if(perspective?.startsWith('bundle.')) {
     return perspective.split('bundle.').at(1)
} else return undefined
```

Reply:
I agree! 😅

<img width="842" alt="Screenshot 2024-07-29 at 12 39 46" src="https://github.com/user-attachments/assets/486d3259-a53d-40ea-8f75-7bee030426a8">